prow/manifests/overlays/metal3/config.yaml

```diff
@@ -28,6 +28,23 @@ plank:
         entrypoint: gcr.io/k8s-prow/entrypoint:v20230329-c93d79fb7d
         initupload: gcr.io/k8s-prow/initupload:v20230329-c93d79fb7d
         sidecar: gcr.io/k8s-prow/sidecar:v20230329-c93d79fb7d
+      resources:
+        clonerefs:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        initupload:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        place_entrypoint:
+          requests:
+            cpu: 10m
+            memory: 10Mi
+        sidecar:
+          requests:
+            cpu: 10m
+            memory: 10Mi
```

Comment:
cant we set this as a default like you are doing for limits in limitrange.yaml ?

Reply:
I'm not understanding. What do you mean by set it as default?
This is setting requests for the sidecar and the numbers are based on the actual usage I saw in the cluster (cpu 1m and memory 9Mi). If we don't set the request here, then it would get the default from the LimitRange (which is way too much for the sidecar)

Reply:
The LimitRange is for all containers. This is for specific prow containers that are named exactly like here. So `sidecar` here is not just any sidecar it is a container that is named `sidecar` and runs together with every prow job. This is how it looks like:

```yaml
apiVersion: v1                                                                                                                                                                                             
kind: Pod                                                                                                                                                                                                  
metadata:                                                                                                                                                                                                                                                                                                                                                                        
  name: a178259f-0b77-11ee-b158-7230da3050f0                                                                                                                                                               
  namespace: test-pods                                                                                                                                                                                                                                                                                                                                                
spec:                                                                                                                                                                                                                                                                                                                                                                     
  containers:                                                                                                                                                                                              
  - name: test
    ...
    # Resources here would come from the LimitRange unless we specify something in the config
  - name: sidecar   # <------ This is the sidecar
    resources:
      requests:
        cpu: 10m
        memory: 10Mi
  initContainers:
  - name: clonerefs
    ...
  - name: initupload
    ...
  - name: place-entrypoint
    ...
```

Reply:
oh ok, my bad, thanks for explaining, I was under the impression it is type and we could set it for all like we are doing in limits.